README.md

```diff
@@ -70,4 +78,6 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+[api_documentation]: https://aka.ms/java-docs
+[samples_readme]: https://aka.ms/java-docs
```

Comment:
Should `samples_readme` be a different link than the `api_documentation` link?

Reply:
Yes, all language repos are supposed to be adding a /samples/README.md to the root of the repo.  I assume this is coming in a follow on PR

Reply:
I am removing this link. Not needed.